app/controllers/builders/SummaryDirectorDetailsSectionBuilder.scala

```diff
@@ -16,86 +16,73 @@
 
 package controllers.builders
 
-import models.api._
+import features.officer.models.view.LodgingOfficer
 import models.view.{SummaryRow, SummarySection}
 
-case class SummaryDirectorDetailsSectionBuilder(vatLodgingOfficer: Option[VatLodgingOfficer] = None)
+case class SummaryDirectorDetailsSectionBuilder(vatLodgingOfficer: LodgingOfficer)
   extends SummarySectionBuilder {
 
   override val sectionId: String = "directorDetails"
 
   val completionCapacity: SummaryRow = SummaryRow(
     s"$sectionId.completionCapacity",
-    vatLodgingOfficer.flatMap(_.name.map(_.asLabel)).getOrElse(""),
-    Some(controllers.vatLodgingOfficer.routes.CompletionCapacityController.show())
+    vatLodgingOfficer.completionCapacity.flatMap(_.officer).map(_.name.asLabel).getOrElse(""),
+    Some(features.officer.controllers.routes.OfficerController.showCompletionCapacity())
   )
 
 
   val formerName: SummaryRow = SummaryRow(
     s"$sectionId.formerName",
-    vatLodgingOfficer.flatMap(_.changeOfName.map(_.nameHasChanged)).collect {
-      case true => vatLodgingOfficer.flatMap(
-        _.changeOfName
-          .flatMap(_.formerName
-            .map(_.formerName))).getOrElse("")
-    }.getOrElse(s"pages.summary.$sectionId.noFormerName"),
-    Some(controllers.vatLodgingOfficer.routes.FormerNameController.show())
+    vatLodgingOfficer.formerName.flatMap(_.formerName).getOrElse(s"pages.summary.$sectionId.noFormerName"),
+    Some(features.officer.controllers.routes.OfficerController.showFormerName())
   )
   val formerNameDate: SummaryRow = SummaryRow(
     s"$sectionId.formerNameDate",
-    vatLodgingOfficer.flatMap(_.changeOfName.map(_.nameHasChanged)).collect {
-      case true => vatLodgingOfficer.flatMap(
-        _.changeOfName
-          .flatMap(_.formerName
-            .flatMap(_.dateOfNameChange
-              .map(_.format(presentationFormatter))))).getOrElse("")
-    }.getOrElse(""),
-    Some(controllers.vatLodgingOfficer.routes.FormerNameDateController.show())
+    vatLodgingOfficer.formerNameDate.map(_.date.format(presentationFormatter)).getOrElse(""),
+    Some(features.officer.controllers.routes.OfficerController.showFormerNameDate())
   )
 
   val dob: SummaryRow = SummaryRow(
     s"$sectionId.dob",
-    vatLodgingOfficer.flatMap(_.dob.map(_.format(presentationFormatter))).getOrElse(""),
-    Some(controllers.vatLodgingOfficer.routes.OfficerSecurityQuestionsController.show())
+    vatLodgingOfficer.securityQuestions.map(_.dob.format(presentationFormatter)).getOrElse(""),
```

Comment:
being picky, these arent actually option by the time you get to the summary (dob nino etc) but its how we loaded the summary before.. and i think AT's will be affected if we didnt do the getOrElse